google_analytics/utils.py

```diff
@@ -60,6 +60,8 @@ def build_ga_params(
 
     # determine the referrer
     referer = referer or request.GET.get('r', '')
+    if referer == domain:
+        referer = ''
```

Comment:
The solution is also not to make referrer blank. For the case where the domain is the same, we should pass through the relative path e.g `/home/` instead of `http://test.com/home/`